packages/datadog-plugin-couchbase/src/index.js

```diff
@@ -0,0 +1,312 @@
+'use strict'
+
+const Tags = require('../../../ext/tags')
+const Kinds = require('../../../ext/kinds')
+const analyticsSampler = require('../../dd-trace/src/analytics_sampler')
+const tx = require('../../dd-trace/src/plugins/util/tx.js')
+
+function startQuerySpan (tracer, config, queryType, query) {
+  const childOf = tracer.scope().active()
+  const span = tracer.startSpan('couchbase.call', {
+    childOf,
+    tags: {
+      'db.type': 'couchbase',
+      'span.type': 'sql',
+      'component': 'couchbase',
+      'service.name': config.service || `${tracer._service}-couchbase`,
+      'resource.name': query,
+      'query.type': queryType,
```

Comment:
Since we're limiting the scope to just N1QL queries, we can just leave this off for now. But I agree, we should have a different span for each kind of query otherwise we're mixing usages.